IFComp/root/src/_current_entry_row.tt

```diff
@@ -52,11 +52,9 @@
         [%# END %]
         
         [% IF entry.platform == 'parchment' 
-              || entry.platform == 'inform'
               || entry.platform == 'website'
-              || entry.platform == 'html' 
               || entry.platform == 'inform-website'
-	      || entry.platform == 'quixe2'
+              || entry.platform == 'quixe2'
```

Comment:
Because we rebuild the platform after translating inform games to parchment games, we shouldn't check for inform here, and separately the html platform is no longer used (in favor of website)